packages/mock-block-dock/src/mock-block-dock.tsx

```diff
@@ -179,7 +186,7 @@ export const MockBlockDock: FunctionComponent<MockBlockDockProps> = ({
     }
   }, [graphService, graphServiceCallbacks]);
 
-  const Component = (
+  const wrappedBlockRenderer = (
```

Comment:
`Component` (with capital letter) is confusing because it implies usage of `<Component />`. Renaming the local var reduces confusion.